pkg/db/kvstore/consulclient.go

```diff
@@ -76,7 +76,7 @@ func (c *ConsulClient) List(ctx context.Context, key string) (map[string]*KVPair
 	deadline, _ := ctx.Deadline()
 	kv := c.consul.KV()
 	var queryOptions consulapi.QueryOptions
-	queryOptions.WaitTime = GetDuration(deadline.Second())
```

Comment:
Not the correct approach. When a int value representing time is converted to Duration, it should be multiplied by base value representing that time unit e.g. time.Second here